fiatconnect-api.md

```diff
@@ -2227,12 +2230,67 @@ A KYC schema containing personal data about a user, as well as documents such as
 	},
 	phoneNumber: `string`,
 	selfieDocument: `string`,
-	identificationDocument: `string`
+  identificationDocument: `string`
```

Comment:
nit: indentation (I don't think I can add tab characters to suggestions)

Reply:
Should be fixed, the file is not consistent as a whole though, sometime there is use of tab, sometime there is double space: 
![image](https://user-images.githubusercontent.com/6547517/215056788-1f918ee9-4e2d-41a0-aaae-90e6ff6e39d8.png)

and 

![image](https://user-images.githubusercontent.com/6547517/215056951-73e15853-69fd-47c5-930b-e1fd8965fb46.png)